desc/objectives/_geometry.py

```diff
@@ -227,6 +233,10 @@ def build(self, eq=None, use_jit=True, verbose=1):
 
         self._profiles = get_profiles(self._data_keys, eq=eq, grid=grid)
         self._transforms = get_transforms(self._data_keys, eq=eq, grid=grid)
```

Comment:
So these have both attributes and copies of the reference in the constants attribute?